libs/src/utils/types.ts

```diff
@@ -1,2 +1,34 @@
 export type Maybe<T> = T | undefined
 export type Nullable<T> = T | null
+
+export interface Logger {
+  /**
+   * Write a 'log' level log.
+   */
+  log: (message: any, ...optionalParams: any[]) => any
```

Comment:
return type `void` instead of `any` i believe? basically means doesn't return anything and is a pure side-effect

Reply:
Are there any loggers that do return something from `log()`? Is sounds weird but I put any just to cover everything. I can change it if I need to

Reply:
i don't think so, but any is fine for now too, no worries